lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helps parsing css selector.
+      module CssSelector
+        COMMON_OPTIONS = %w[
+          above below left_of right_of near count minimum maximum between text
+          id class style visible obscured exact exact_text normalize_ws match
+          wait filter_set focused
+        ].freeze
+
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   id?('#some-id') # => true
+        #   id?('.some-class') # => false
+        def id?(selector)
+          selector.start_with?('#')
+        end
+
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   attribute?('[attribute]') # => true
+        #   attribute?('attribute') # => false
+        def attribute?(selector)
+          selector.start_with?('[')
+        end
+
+        # @param selector [String]
+        # @return [Array<String>]
+        # @example
+        #   attributes('a[foo-bar_baz]') # => {"foo-bar_baz=>true}
+        #   attributes('button[foo][bar]') # => {"foo"=>true, "bar"=>true}
+        #   attributes('table[foo=bar]') # => {"foo"=>"'bar'"}
+        def attributes(selector)
+          selector.scan(/\[(.*?)\]/).flatten.map do |attr|
+            key, value = attr.split('=')
+            [key, normalize_value(value)]
+          end.to_h
```

Comment:
```suggestion
          selector.scan(/\[(.*?)\]/).flatten.to_h do |attr|
            key, value = attr.split('=')
            [key, normalize_value(value)]
          end
```
`to_h` accepts a block to construct the key/value pair, no need of map.to_h

Reply:
It is certainly better that way. Thank you very much.